redash/query_runner/athena.py

```diff
@@ -43,7 +43,7 @@ def format(self, operation, parameters=None):
 
 class Athena(BaseQueryRunner):
     noop_query = 'SELECT 1'
-    sample_query = "SELECT * FROM {table} LIMIT 1"
+    sample_query = "SELECT * FROM {table} TABLESAMPLE SYSTEM (1) LIMIT 1"
```

Comment:
Do you mind adding a comment to this and the Presto runner explaining what the `TABLESAMPLE SYSTEM (1)` means?